cmd/lncli/invoicesrpc_active.go

```diff
@@ -189,6 +184,14 @@ var addHoldInvoiceCommand = cli.Command{
 				"private channels in order to assist the " +
 				"payer in reaching you",
 		},
+		cli.StringFlag{
+			Name:  "hash",
+			Usage: "the invoice hash",
+		},
+		cli.StringFlag{
```

Comment:
Not sure I follow a use-case where we already have the pre-image, yet want to always make it a hold invoice. Also need to cross reference this with the latest thinking around AMP variants.

Reply:
Checking webshop inventory to see if the goods are still in stock?